scripts/CVE-2018-13379.nse

```diff
@@ -0,0 +1,78 @@
+description = [[
+Performs a scan to check whether the scanned server is vulnerable to CVE-2018-13379
+
+CVE-2018-13379:
+An Improper Limitation of a Pathname to a Restricted Directory ("Path Traversal") in Fortinet FortiOS 6.0.0 to 6.0.4, 5.6.3 to 5.6.7 under SSL VPN web portal allows an unauthenticated attacker to download system files via special crafted HTTP resource requests.
+]]
+---
+-- @usage
+-- nmap --script CVE-2018-13379 -p <port> <host>
+-- @output
+-- PORT   STATE SERVICE
+-- 443/tcp open  http
+-- | CVE-2018-13379: 
+-- |   Host is vulnerable to CVE-2018-13379
+-- @changelog
+-- 2019-23-08 - Author Alejandro Flores Covarrubias <alejandro.florescova@gmail.com> by Purple Security
+-- Twitter: alejandrocovrr
+-- Twitter: purplesecmx
+
+author = "Alejandro Flores Covarrubias <alejandro.florescova@gmail.com> @alejandrocovrr @purplesecmx"
+categories = {"discovery", "intrusive","vuln"}
+
+local http = require "http"
+local stdnse = require "stdnse"
+local shortport = require "shortport"
+
```

Comment:
Make sure all used libraries are required first. You are missing the table library here.